### ISSUE TITLE
Lunar Lander hyperparameters

### ISSUE THREAD

**Issue by richielo** [nikhilbarhate99/TD3-PyTorch-BipedalWalker-v2#4]
May I ask what are your hyperparameters for lunar lander?

**Comment by nikhilbarhate99** [comment]
######### Hyperparameters #########
    env_name = "LunarLanderContinuous-v2"
    random_seed = 0
    log_interval = 10           # print avg reward after interval
    solved_reward = 200          # stop training if avg_reward > solved_reward
    save_episode = 500         # keep saving after n episodes
    max_episodes = 10000        # max num of episodes
    max_timesteps = 500         # max timesteps in one episode
    
    gamma = 0.99                # discount for future rewards
    batch_size = 100            # num of transitions sampled from replay buffer
    lr = 0.001
    exploration_noise = 0.1 
    polyak = 0.995              # target policy update parameter (1-tau)
    policy_noise = 0.2          # target policy smoothing noise std
    noise_clip = 0.5
    policy_delay = 2            # delayed policy updates parameter
    
    directory = "./"            # save trained models
    filename = "TD3_{}_{}".format(env_name, random_seed)
    ###################################
